feat: add documentation for new Azure entities supported by Azure Monitor

glennthomas1 opened the pull request:
<!-- Thanks for contributing to our docs! -->

<!-- For Japanese readers: 
もしドキュメントの日本語訳で問題を見つけた場合はPRではなくissueを提出してください。
日本語訳へのPRについてはまだ取り込む準備ができていません。-->

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
  - releasing documentation for newly added Azure entities supported by Azure Monitor integration
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/newrelic/docs-website?pullRequest=12771) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/newrelic/docs-website?pullRequest=12771) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/newrelic/docs-website?pullRequest=12771) it.</sub>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/newrelic/docs-website?pullRequest=12771) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/newrelic/docs-website?pullRequest=12771) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/newrelic/docs-website?pullRequest=12771) it.</sub>

glennthomas1 commented:
It looks like the Gatsby Build Service timed out - can you tell me if there is something I need to correct to get it to go through?

glennthomas1 commented:
Updated for discussed changes. Removed one improperly uploaded file. I will create another PR for this one with the title: Project NorthStar Azure - adding one additional cohort 1 doc. Please process these two PR's together. I'll update again with the PR number.
